docs/site/Community-extensions.md

```diff
@@ -67,6 +67,10 @@ list.
       <td><a href="https://www.npmjs.com/package/@sourceloop/audit-log">@sourceloop/audit-log</a></td>
       <td>Implement audit logs in loopback applications for all DB transactions</td>
     </tr>
+    <tr>
+      <td><a href="https://www.npmjs.com/package/loopback-rabbitmq">loopback-rabbitmq</a></td>
+      <td>An Rabbitmq extension that implements Queue consumers and producers.</td>
```

Comment:
Just nitpick. Since other entries don't have `.` at the end, I'd suggest to keep it consistent. 

```suggestion
      <td>An Rabbitmq extension that implements Queue consumers and producers</td>
```

Reply:
@dhmlau sorry for my lack of attention, thanks for the fix